config/manager/priority_class.yaml

```diff
@@ -0,0 +1,7 @@
+apiVersion: scheduling.k8s.io/v1
+kind: PriorityClass
+metadata:
+  name: priority-class
```

Comment:
This class is never applied, right? We should use it for the operator and the otel collectors (trace-collector-priority-class and metric-gateway-priority-class)